src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -381,54 +388,71 @@ public void Stake(WalletSecret walletSecret)
         {
             this.logger.LogTrace("()");
 
-            if (Interlocked.CompareExchange(ref this.stakeProgressFlag, StakeInProgress, StakeNotInProgress) == StakeInProgress)
+            if (Interlocked.CompareExchange(
+                    ref this.stakeStateFlag,
```

Comment:
Do you really prefer this way of calling methods with each argument separated line or is it just the IDE setting that is doing that for you?

In any case, if you really want to have it on multiple lines, which I think is terrible ugly and less readable, you should respect the indentation which is four spaces.

Reply:
Yes I do. It makes it much easier to see what is passed into the method. There are too many brackets, commas and operators in one line to clearly see boundaries of function and comparison. "Ugly" is highly subjective and comes down to personal preference. I would prefer to leave it as it. I have fixed tabs.

Reply:
I still see eight spaces.

I agree that it is subjective, that is why I don't require you to write it in the other way is there is no policy for that. Though I really want you to know that it is ugly ;-)

Reply:
That’s correct. 2 indents. Logically Parameters should not be on the same line as method. So Interlocked should not be online with ref

Reply:
Though you should not be propagating your personal style at expense of creating code that is inconsistent with the rest of the code.

Reply:
OK, https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/coding-style.md first paragraph